res/css/views/elements/_ImageView.scss

```diff
@@ -44,6 +44,8 @@ $button-gap: 24px;
     display: flex;
     justify-content: space-between;
     align-items: center;
+    animation-name: mx_ImageView_panel_keyframes;
```

Comment:
Please handle @media (prefers-reduced-motion), so that we don't animate things when user doesn't want reduced motion.